src/entity/user/UserTopReader.ts

```diff
@@ -4,6 +4,9 @@ import type { Keyword } from '../Keyword';
 
 @Entity()
 @Index('IDX_user_top_reader_userId_issuedAt', { synchronize: false })
+@Index('IDX_user_top_reader_userId_issuedAt_keywordValue', {
+  synchronize: false,
+})
```

Comment:
```suggestion
@Index(
  'IDX_user_top_reader_userId_issuedAt_keywordValue',
  ['userId', 'issuedAt', 'keywordValue'],
  { unique: true },
)
```

this would be equivalent?

Reply:
Yup